pkg/multiTenancyPlugins/flavors/flavors.go

```diff
@@ -84,11 +85,39 @@ func (flavorsImpl *DefaultFlavorsImpl) Handle(command utils.CommandEnum, cluster
 	if reqBody, _ := ioutil.ReadAll(r.Body); len(reqBody) > 0 {
 		var flavorIn Flavor
 		var buf bytes.Buffer
-		var containerConfig dockerclient.ContainerConfig
-		if err := json.NewDecoder(bytes.NewReader(reqBody)).Decode(&containerConfig); err != nil {
-			return err
```

Comment:
Why this was replaced with
var reqBody []byte
		defer r.Body.Close()
		if reqBody, _ = ioutil.ReadAll(r.Body); len(reqBody) > 0 {
			if err := json.NewDecoder(bytes.NewReader(reqBody)).Decode(&oldconfig); err != nil {
				log.Error(err)
				return err
			}
  		}